cellxpredict/session.py

```diff
@@ -11,6 +11,9 @@ def write_config_json_file(config: ConfigBase) -> None:
     # extract the params into dict:
     json_data = {prm : str(getattr(config, prm)) for prm in config.__dict__}
 
+    src_files = list(Path(config.src_dir).glob("*.tfrecord"))
```

Comment:
Looks good, thanks @chris-soelistyo!

Minor suggestion, could you add sorting because I've noticed before that `glob` doesn't automatically follow alphabetical order, making it sometimes a bit counterintuitive 
`list(sorted(Path(config.src_dir).glob("*.tfrecord")))`